tools/pot/openvino/tools/pot/api/README.md

```diff
@@ -90,10 +30,15 @@ class openvino.tools.pot.Metric()
 ```
 An abstract class representing an accuracy metric.
 
-All subclasses should override the following properties:
-- `value` - returns the accuracy metric value for the last model output.
-- `avg_value` - returns the average accuracy metric value for all model outputs.
-- `attributes` - returns a dictionary of metric attributes:
+All instances should override the following properties:
+- `value` - returns the accuracy metric value for the last model output in a format of `Dict[str, numpy.array]`.
+- `avg_value` - returns the average accuracy metric over collected model results in a format of `Dict[str, numpy.array]`.
+- `higher_better` should return `True` if a higher value of the metric corresponds to a better performance, otherwise, returns `False`. Default implementation returns `True`.
+
+and methods:
+- `update(output, annotation)` - calculates and updates the accuracy metric value using last model output and annotation. The model ouput and annotation should be passed in this method. It should also contain the model specific post-processing in case if the model returns the raw output.
+- `reset()` - resets collected accuracy metric. 
+- `get_attributes()` - returns a dictionary of metric attributes:
```

Comment:
```suggestion
- `update(output, annotation)` - calculates and updates the accuracy metric value using the last model output and annotation. The model output and annotation should be passed in this method. It should also contain the model-specific post-processing in case the model returns the raw output.
- `reset()` - resets collected accuracy metric. 
- `get_attributes()` - returns a dictionary of metric attributes:
```